client/src/routes/playlists/playlist.component.tsx

```diff
@@ -0,0 +1,89 @@
+import { InputSerializationFilterSensitiveLog } from '@aws-sdk/client-s3';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useSelector } from 'react-redux';
+import { selectCurrentUser } from '../../store/user/user.selector'
+
+type Video = {
+  title: string,
+  thumbnail: string,
+  description: string,
+  videoId: string
+}
+
+const PlayList = () => {
+  const [currentSearch, setCurrentSearch] = useState('');
+  const [videos, setVideos] = useState([]);
+  const [playlist, setPlaylist] = useState<Video[]>([]);
+  const currentUser = useSelector(selectCurrentUser);
+
+
+  const handleSearchChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setCurrentSearch(e.target.value);
+  }
+  const handleSubmit = async () => {
+    try {
+      const { data } = await axios.get(`/api/playlists/search/${currentSearch}`)
+      setVideos(data);
+    }
+    catch(err) {
+      console.error(err);
+    }
+  }
+
+  const handleSaveToPlaylist = async (video: Video) => {
+    const { data } = await axios.post('/api/playlists/add', {video, userId: currentUser.id})
+    setPlaylist([...playlist, data])
+  }
+
+  console.log('playlist', playlist);
+  return (
+    <div className="container" >
+    <div className='row'>
+      
+    <div className='col'>
+
+  
+    <h2>Search for Music (scary) </h2>
+    <input value={currentSearch}  onChange={handleSearchChange} placeholder='type for songs....'/>
+    <button onClick={handleSubmit} >BOO!</button>
+
+    { videos.map((video: Video) => {
+      return (
+        <div key={video.videoId} className="card" style={{width: '18rem', backgroundColor:'black'}}>
+        {/* <video src="video.mp4"></video> */}
```

Comment:
Unsure what the point of this commented video element is? playlist component line 54

Reply:
Same as 72